clippy_lints/src/strings.rs

```diff
@@ -173,6 +173,7 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for StringLitAsBytes {
                             );
                         } else if callsite == expanded
                             && lit_content.as_str().chars().all(|c| c.is_ascii())
+                            && lit_content.as_str().len() <= 32
```

Comment:
It would be nice to extract the number into a const and add a small comment to it.

Reply:
How about 7d1a9447?